Packs/AzureSecurityCenter/Integrations/AzureSecurityCenter_v2/AzureSecurityCenter_v2.yml

```
@@ -201,6 +202,12 @@ script:
       name: alert_id
       required: true
       secret: false
+    - default: false
+      description: Subscription ID.
```

Let's add that in case the subscription_id was given both as a parameter and as a command argument - the command argument will be used.

---

it is already there, did you mean something else?

---

Add it to the argument's description as well.
```suggestion
      description: Subscription ID.  If empty, the subcription_id from the instance configuration is used.
```